extensions/amp-story/1.0/progress-bar.js

```diff
@@ -119,10 +177,180 @@ export class ProgressBar {
       true /** callToInitialize */
     );
 
+    this.storeService_.subscribe(
+      StateProperty.RTL_STATE,
+      rtlState => {
+        this.onRtlStateUpdate_(rtlState);
+      },
+      true /** callToInitialize */
+    );
+
+    const ampdoc = Services.ampdocServiceFor(this.win_).getSingleDoc();
+    Services.viewportForDoc(ampdoc).onResize(
+      debounce(this.win_, () => this.onResize_(), 300)
+    );
+
+    this.barWidthPx_ = this.storyEl_
+      .querySelector('amp-story-page')
+      .getBoundingClientRect()./*OK*/ width;
+
+    this.vsync_.mutate(() => {
+      // Don't animate the progress bar in pageload.
+      this.timer_.delay(() => {
+        this.root_.classList.add('i-amphtml-animate-progress');
+      }, 60);
```

Comment:
60ms seems very arbitrary, we should remove this, cf the other comment on the `render_` method.